docker/composes/docker-compose-dev.yml

```diff
@@ -26,7 +26,9 @@ services:
       context: ./modules/back-end
       dockerfile: ./deploy/Dockerfile
     environment:
-      - SSOEnabled=true
+      - SSOEnabled=false
```

Comment:
I think we should keep `SSOEnabled` to `true`

Reply:
fixed with (commit #16fc07)[16fc07608ca6150c363575b4f0a8bce5c8f5cb71]